Files/Filesystem/ListedItem.cs

```diff
@@ -180,6 +180,12 @@ public DateTimeOffset ItemDateModifiedReal
             set
             {
                 ItemDateModified = GetFriendlyDateFromFormat(value, DateReturnFormat);
+
+                FolderTooltipText = string.Format("Name: {0}\nType: {1}\nDate modified: {2}",
```

Comment:
First, it is not a Folder tooltip, it is Item tooltip. Second, why did you placed it inside ItemDateModifiedReal setter?

Reply:
Third, for some reason, for me Type field is always empty.

Reply:
> First, it is not a Folder tooltip, it is Item tooltip. Second, why did you placed it inside ItemDateModifiedReal setter?

First as you can see on [#721](https://github.com/files-community/Files/issues/721)  images both of them looks as tooltips

Second, that tooltip only should be setted only if 'date modified' be defined

Third, i don't know why, but i suggest you to check if ItemType is correctly setted. As you can see on my previous tooltip image file type is well loaded.

Reply:
I mean name of the property should be changed to `ItemTooltipText`, because tooltip is displayed not only for folders.

Reply:
By the way, can you add translation for new string you added?

Reply:
> By the way, can you add translation for new string you added?

I already tried using:

- "nameColumn".GetLocalized();
- "dateColumn".GetLocalized();
- "typeColumn".GetLocalized();

But it's not well loaded into tooltip.
However, i will try a different approach.

-

Reply:
Just add `Name: {0}\nType: {1}\nDate modified: {2}` string to Strings/en-US/Resources.resw file and use it in the code via GetLocalized()